tests/gold_tests/transaction/txn.test.py

```diff
@@ -52,15 +52,15 @@
 })
 
 ts.Disk.remap_config.AddLine(
-    'map http://127.0.0.1:{0} http://127.0.0.1:{1}'.format(
+    'map http://txn.test:{0} http://127.0.0.1:{1}'.format(
```

Comment:
Another remap rule [rejected](https://github.com/apache/trafficserver/blob/8d01164257417434b667021365b5f07d0ec596e0/tests/tools/sessionvalidation/sessionvalidation.py#L200-L203) by the microserver, preventing the health check.